plugins/generator-1.15.2/forge-1.15.2/templates/gui.java.ftl

```diff
@@ -584,6 +584,28 @@ import ${package}.${JavaModName};
 		<#assign btid = 0>
         <#list data.components as component>
 			<#if component.getClass().getSimpleName() == "Button">
+			<#if component.externalLink?length gt 0>

```

Comment:
This is not valid freemarker syntax, also there are better checks for string lenth I think

Reply:
Read the end of this page. It's explained we can use `gt` to replace `>`. https://freemarker.apache.org/docs/ref_directive_if.html